src/components/Header/Header.js

```diff
@@ -49,16 +53,28 @@ function Header() {
               className={`dropdown-menu ${show ? 'show' : ''}`}
               aria-labelledby="dropdown01"
             >
-              <Link className="dropdown-item" to="/orders">
+              <Link
```

Comment:
these are pretty much the same all the way through. Can we pull out into an array const and loop over to create this dropdown?

Reply:
Not a problem!